index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>William/Dennis PAGE</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center">William and Dennis Collaboration Code</h1>
+        <p id="description" class="description text-center">
+          This is a survey for our code...
+        </p>
+      </header>
+      <form id="survey-form">
+        <div class="form-group">
+          <label id="name-label" for="username">username</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            class="form-control"
+            placeholder="Enter your username"
+            required
+          />
+        </div>
+        <div class="form-group">
```

Comment:
- [ ] I think it would be better if you used suggestive phrases instead of using words that read as an order when requesting required changes so that the student gives emphasis to the changes requested. We don't want the student to feel like we are giving them orders so they feel open to accepting our feedback